spec/controllers/bulk_ingest_controller_spec.rb

```diff
@@ -60,38 +60,62 @@
   end
 
   describe "POST #browse_everything_files" do
+    let(:attributes) do
+      {
+        workflow: { state: "pending" },
+        collections: ["1234567"],
+        visibility: "open",
+        mvw: false,
+        selected_files: selected_files
+      }
+    end
+    let(:selected_files) do
+      {
+        "0" => {
+          "url" => "file:///base/4609321/1.tif",
+          "file_name" => "1.tif",
+          "file_size" => "100"
+        }
+      }
+    end
+
     before do
       allow(IngestFolderJob).to receive(:perform_later)
       allow(IngestFoldersJob).to receive(:perform_later)
     end
 
     context "with one single-volume resource" do
       before do
-        stub_bibdata(bib_id: 4609321)
+        stub_bibdata(bib_id: "4609321")
       end
 
-      let(:attributes) do
-        {
-          workflow: { state: "pending" },
-          collections: ["1234567"],
+      it "ingests the directory as a single resource" do
+        post :browse_everything_files, params: { resource_type: "scanned_resource", **attributes }
+        expected_attributes = {
+          directory: "/base/4609321",
+          state: "pending",
           visibility: "open",
-          mvw: false,
-          selected_files: selected_files
+          member_of_collection_ids: ["1234567"],
+          source_metadata_identifier: "4609321"
         }
+        expect(IngestFolderJob).to have_received(:perform_later).with(hash_including(expected_attributes))
       end
-      let(:selected_files) do
-        {
-          "0" => {
-            "url" => "file:///base/4609321/1.tif",
-            "file_name" => "1.tif",
-            "file_size" => "100"
-          }
-        }
+    end
+
+    context "when the directory looks like a bibid, but isn't valid" do
```

Comment:
This one is a replacement for the "June 31" test that was removed in the other spec?

Reply:
Yes, though the scenario is a little different (invalid bibid vs. directory name that doesn't match the pattern).  I'll add an example that uses "June 31" like the old test.